codalab/apps/health/views.py

```diff
@@ -34,7 +39,8 @@ def get_health_metrics():
         if (job.updated - job.created) > timedelta(minutes=10):
             jobs_lasting_longer_than_10_minutes.append(job)
 
-    jobs_failed = Job.objects.filter(status=Job.FAILED).order_by("-updated")[:10]
+    jobs_failed = Job.objects.filter(
+        status=Job.FAILED).order_by("-updated")[:10]
```

Comment:
this the the prev 3 changes could used revered or updated no more `\` crazyness. lol line 29 while long made sense before the split